doc/how-to-release.md

```diff
@@ -139,6 +139,15 @@ deploy:
       tags: true
 ```
 
+The secure `api_key` should contain `tobiiprotools`'s Github token 
+(can be found in the **designated safe location**) and must be encrypted with Travis CLI like 
+
+```shell
+travis encrypt <GH_TOKEN>
```

Comment:
I'm not sure if it's needed now that they have "merged" .com and .org but I use the "--pro" flag too

Reply:
Seems like it is still needed, can be checked with `> travis endpoint`. Btw, travis CLI hasn't seen a single commit since November :(